go/libkb/sig_chain.go

```diff
@@ -370,6 +378,29 @@ func (sc *SigChain) VerifyChain(m MetaContext) (err error) {
 		curr.chainVerified = true
 	}
 
+	newHPrevInfo := NewInitialHPrevInfo()
```

Comment:
The main for loop in VerifyChain (folded away right now) traverses a user's sigchain backwards, I can't see any reason for that. HPrevInfo computing code needs to traverse forwards, so I can save a pass if I can reverse the traversal in the first for loop in VerifyChain.

Reply:
maybe it use to go backwards to verify the start of the current subchain.